src/__tests__/index.js

```diff
@@ -0,0 +1,146 @@
+import { loadFixtures, unloadFixtures } from 'util/fixtures';
+import client from 'util/client';
+import { apolloServer } from '../index';
+import fixtures from '../__fixtures__/index';
+import { createTestClient } from 'apollo-server-testing';
+import MockDate from 'mockdate';
+
+jest.mock('koa', () => {
+  const Koa = jest.requireActual('koa');
+  Koa.prototype.listen = () => null;
+  return {
+    default: Koa,
+    __esModule: true,
+  };
+});
+
+describe('apolloServer', () => {
+  const actualGraphQLServerOptions = apolloServer.graphQLServerOptions;
+  const mockGraphQLServerOptions = ctx => async () =>
+    actualGraphQLServerOptions.call(apolloServer, { ctx });
+  let now;
+
+  beforeAll(async () => {
+    MockDate.set(1602288000000);
+    now = new Date().toISOString();
+
+    apolloServer.app;
+    await loadFixtures(fixtures);
+  });
+
+  afterAll(async () => {
+    MockDate.reset();
+
+    await unloadFixtures(fixtures);
+    await client.delete({
+      index: 'users',
+      type: 'doc',
+      id: '6LOqD_gsUWLlGviSA4KFdKpsNncQfTYeueOl-DGx9fL6zCNeA',
+    });
+  });
+
+  it('resolves current web user', async () => {
+    const appId = 'WEBSITE';
+    const userId = 'testUser1';
+
+    apolloServer.graphQLServerOptions = mockGraphQLServerOptions({
+      appId,
+      userId,
```

Comment:
Is `userId` being sent to `graphQLServerOptions` in real scenarios? When is it written?

Reply:
this is a workaround since `createTestClient` doesn't let you provide `context` so overwriting `integrationContextArgument` for `graphQLServerOptions` is kinda like mimicking getting `appId` and `userId` 
I will add a comment stating that.
https://github.com/apollographql/apollo-server/blob/main/packages/apollo-server-core/src/ApolloServer.ts#L939